### PR TITLE
Readme updated

### PR DESCRIPTION
**Pull request by DyxaDevelop**

### CODE REVIEW

**Comment by DyxaDevelop** [comment]
This PR makes the following changes to the README.md

- Added visual enhancements: logo and various badges (Discord, Twitter, GitHub, etc.).
- Expanded the Table of Contents with new sections and links.
- Updated project description with a focus on Discord OAuth2 integration and user data handling.
- Clarified prerequisites: added PostgreSQL, adjusted Node.js minimum version (14+).
- Expanded setup and run instructions, including dependency installation and environment variables.
- Added a section detailing available scripts (dev, build, start, etc.).
- Provided more detailed deployment steps for Vercel.
- Added a "Troubleshooting" section with common issue resolutions.

File: README.md [hunk 1]
```diff
@@ -1,56 +1,175 @@
...
+<p align="center">
+    <img alt="Monad Discord Auth" src="https://github.com/user-attachments/assets/32f470e7-ca9f-4610-a28b-0d630456dce2">
+</p>
 
-This is a Next.js project that integrates Discord authentication using NextAuth for a seamless login experience with Discord credentials.
+<p align="center">
+    <a href="https://discord.gg/T7KKNT4cng">
+        <img src="https://img.shields.io/badge/Discord-Monad%20Developers-7289DA?logo=discord&logoColor=white" alt="Discord">
+    </a>
+    <a href="https://x.com/monad_xyz">
```
Comment (portdeveloper) [inline README.md:9]: Since this is a community project let's remove the x link for monad here

File: README.md [hunk 2]
```diff
@@ -1,56 +1,175 @@
...
+## 3. Prerequisites
 
-[Deploy with Vercel](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme)
+Ensure that your system meets the following requirements:
 
-## Contributing
+- **Node.js**: Ensure you have Node.js installed (version 14 or later is recommended). You can download and install Node.js from the official website [here](https://nodejs.org/en/).
+- **pnpm** is a package manager that can be installed in various ways. One method is a global installation via npm, which you can do by entering the following in your terminal:
+```bash
+npm install -g pnpm
```
Comment (portdeveloper) [inline README.md:54]: Let's have this as the first option! https://pnpm.io/installation

**Review by portdeveloper — CHANGES_REQUESTED** [review]
Looks good to me except few nitpicks here and there 👍

File: README.md [hunk 3]
```diff
@@ -1,56 +1,175 @@
...
+3. Make your changes and commit them with a clear and descriptive commit message.
+4. Push your changes to your fork and create a pull request to the main repository.
+
+For detailed guidelines, refer to the [CONTRIBUTING.md](CONTRIBUTING.md) file.
+
+---
+
+## 9. License
+
+This project is licensed under the MIT License. See the [LICENSE](LICENSE.md) file for details.
```
Comment (portdeveloper) [inline README.md:175]: Hey I think we don't have a LICENSE file yet, could you please add an MIT license or remove this part?

File: LICENSE [hunk 4]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2024 Monad Developers
```
Comment (portdeveloper) [inline LICENSE:3]: Sorry for being nitpicky again but maybe we should change it to "Monad Developer Community"

**Review by portdeveloper — CHANGES_REQUESTED** [review]
All is great but i have dropped one comment on one line!

File: LICENSE (hunk 4 above)
Comment (DyxaDevelop) [reply LICENSE]: No problem, everything should be perfect!

**Review by portdeveloper — APPROVED** [review]
LGTM! Thanks @DyxaDevelop !!!

**Pull request merged**